fix(docs): normalize table headings (#5227)
hammerandchisel/discord-api-docs: docs/resources/Auto_Moderation.md

Characterizes the type of content which can trigger the rule.

| Name                 | Value   | Description                                                                           |
| -------------------- | ------- | ------------------------------------------------------------------------------------- |

> | KEYWORD              | 1       | check if content contains words from a user defined list of keywords. Max per guild 3 |

Reviewer: ```suggestion
| KEYWORD              | 1       | check if content contains words from a user defined list of keywords. Max 3 per guild |
```

> | HARMFUL_LINK         | 2       | check if content contains any harmful links. Max per guild 1                          |

Reviewer: ```suggestion
| HARMFUL_LINK         | 2       | check if content contains any harmful links. Max 1 per guild                          |
```

> | SPAM                 | 3       | check if content represents generic spam. Max per guild 1                             |

Reviewer: ```suggestion
| SPAM                 | 3       | check if content represents generic spam. Max 1 per guild                             |
```

> | KEYWORD_PRESET       | 4       | check if content contains words from internal pre-defined wordsets. Max per guild 1   |

Reviewer: ```suggestion
| KEYWORD_PRESET       | 4       | check if content contains words from internal pre-defined wordsets. Max 1 per guild   |
```

###### Trigger Metadata

Additional data used to determine whether a rule should be triggered. Different fields are relevant based on the
value of [trigger_type](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-rule-object-trigger-types).

| Field          | Type                                                                                                              | Description                                                                                                    |
| -------------- | ----------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------- |

> | keyword_filter | array of strings *                                                                                                | Associated trigger `KEYWORD`. Substrings which will be searched for in content                                 |

Reviewer: ```suggestion
| keyword_filter | array of strings *                                                                                                | substrings which will be searched for in content by `KEYWORD` triggers                                |
```

> | presets        | array of [keyword preset types](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-rule-object-keyword-preset-types) | Associated trigger `KEYWORD_PRESET`. The internally pre-defined wordsets which will be searched for in content |

Reviewer: ```suggestion
| presets        | array of [keyword preset types](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-rule-object-keyword-preset-types) | the internally pre-defined wordsets which will be searched for in content by `KEYWORD_PRESET` triggers |
```

> | allow_list     | array of strings *                                                                                                | Associated trigger `KEYWORD_PRESET`. Substrings which will be exempt from triggering the preset trigger type   |

Reviewer: ```suggestion
| allow_list     | array of strings *                                                                                                | substrings which will be exempt from triggering the preset trigger type  |
```

KEYWORD_PRESET is already implied

Additional data used when an action is executed. Different fields are relevant based on the
value of [action type](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-action-object-action-types).

| Field            | Type       | Description                                                                           |
| ---------------- | ---------- | ------------------------------------------------------------------------------------- |
| channel_id       | snowflake  | Associaed action `SEND_ALERT_MESSAGE`. Channel to which user content should be logged |

> | duration_seconds | integer    | Associaed action `TIMEOUT`. Timeout duration in seconds *                             |

Reviewer: ```suggestion
| channel_id       | snowflake  | Associated action `SEND_ALERT_MESSAGE`. Channel to which user content should be logged |
| duration_seconds | integer    | Associated action `TIMEOUT`. Timeout duration in seconds *                             |
```

This unfortunately doesn't work in English; and same as above with others, it removes machine readability as well
